writer.go

```diff
@@ -40,10 +50,17 @@ func (w *Writer) Run() {
 				time.Sleep(2 * time.Second)
 				continue
 			}
+
 			// InfluxDB docs say that only 204 exactly is truly successful,
 			// and in fact 200 OK is not successful. Strange, but okay...
 			if resp.StatusCode != 204 {
 				log.Println("backend write returned", resp.Status)
+				if resp.StatusCode == 400 {
```

Comment:
*Should* be the case, but then again, "200 OK" isn't a success according to the above comment, so I was being very limited in terms of how this response is handled. What other Influxdb 4xx errors are you concerned about?